custom/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
   annotationProcessor("com.google.auto:auto-common:0.8")
   implementation("com.google.auto.service:auto-service:1.0-rc3")
   implementation("com.google.auto:auto-common:0.8")
+  implementation("com.google.guava:guava:29.0-android")
```

Comment:
Why this version? [Upstream has](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/8d74baa2e4d186b1e78ebdc88ae0a7f73a297e69/gradle/dependencies.gradle#L15) `"20.0", // Last version to support Java 7`

Reply:
apparently `custom` depends on both `sdk` and `javaagent` and they themselves have different guavas.

Reply:
It's a wonder that it works at all 😂